backend/contentstore/s3store.go

```diff
@@ -70,6 +71,22 @@ func (s *S3Store) Read(key string) (io.Reader, error) {
 	return res.Body, nil
 }
 
+func (s *S3Store) SendURL(key string) (*string, error) {
+	contentType := "image/jpeg"
+	req, _ := s.s3Client.GetObjectRequest(&s3.GetObjectInput{
+		Bucket:              aws.String(s.bucketName),
+		Key:                 aws.String(key),
+		ResponseContentType: aws.String(contentType),
+	})
+
+	url, err := req.Presign(time.Hour * 8)
```

Comment:
do you guys have any thoughts about how long these images should persist?

Reply:
maybe 5-10 minutes max. The browser is going to download these immediately and each request is going to create a new pre-signed url so there is no reason these need to be alive for very long. Two questions:
1. Can we limit the amount of times the evidence can be requested (1 or 2 is probably the right amount)
2. When you pop open the modal does it load a different version of the image or does it just render a scaled up version of the one that is already downloaded? If it's a new version are we requesting a new signed URL and downloading it?

Reply:
oh good point 👍 
1. As in to limit the number of times a user can reload the page in a certain time frame?
2. oooh good question - I hadn't thought about that. It is two requests with a new signed url. I can look into ways around that if you'd like

Reply:
When you change your view in the timeline, like running a filter, you get new URLs to retrieve the images again. I think the most time someone would spend scrolling through a view would be around 30minutes or so? We could start there and increase if needed.

Reply:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html indicates that pre-signed urls are intended to be multiple use, so limiting the time window they're active will be our best option.